lib/application.js

```diff
@@ -300,22 +300,34 @@ app.engine = function(ext, fn){
  * See the Router#param() docs for more details.
  *
  * @param {String|Array} name
- * @param {Function} fn
+ * @param {Function|Array} fn
```

Comment:
This jsdoc does not reflect what you are doing. It documents the call as .param('id', [fn1, fn2])